### ISSUE TITLE
Alt text missing equal sign

### ISSUE THREAD

**Issue by farishkash** [smaddox42/capstone#4]
https://github.com/smaddox42/capstone/blob/a581585eec16b9a983420471bf61fe6c10fd9c22/index.html#L173

Missing the equal sign in some of these, should be

`alt = "Person designing with phone and paper"/> `
